includes/scripts.php

```diff
@@ -432,3 +432,35 @@ function give_admin_hide_notice_shortly_js() {
 }
 
 add_action( 'admin_head', 'give_admin_hide_notice_shortly_js' );
+
+/**
+ * Admin js code
+ *
+ * This code helps to hide php version upgrade nag for 24 hour if admin user dismissed notice.
+ *
+ * @since 1.8.9
+ *
+ * @return void
+ */
+function give_admin_hide_outdated_php_notice_shortly_js() {
```

Comment:
Can you make this output conditionally based on if the notice is shown or not? Otherwise, this code will output in wp-admin even if they are using PHP 5.3+